.circleci/config.yml

```diff
@@ -291,71 +295,72 @@ jobs:
 workflows:
   openlineage:
     jobs:
-      - gradle/test:
-          app_src_directory: client/java
-      - test-client-python
-      - test-integration-common
+      - build-client-java
+      - publish-snapshot-client-java:
+          <<: *only_on_main
+          context: release
+          requires:
+            - build-client-java
       - build-integration-spark
-      - build-java-client
       - integration-test-spark:
           requires:
             - build-integration-spark
-      - test-integration-airflow
-      - integration-test-airflow:
-          requires:
-            - test-integration-airflow
-
-      #publish snapshots
-      - publish-snapshot-java-client:
-          <<: *only-on-main-without-release-tag
-          context: release
-          requires:
-            - build-java-client
       - publish-snapshot-integration-spark:
-          <<: *only-on-main-without-release-tag
+          <<: *only_on_main
           context: release
           requires:
             - integration-test-spark
-
+      - test-client-python
       - build-client-python:
-          <<: *only-on-main-without-release-tag
+          <<: *only_on_main
           build_tag: .dev${CIRCLE_BUILD_NUM}
+          requires:
+            - test-client-python
+      - test-integration-common
       - build-integration-common:
-          <<: *only-on-main-without-release-tag
+          <<: *only_on_main
           build_tag: .dev${CIRCLE_BUILD_NUM}
+          requires:
+            - test-integration-common
+      - test-integration-airflow
+      - integration-test-airflow:
```

Comment:
While we're cleaning up, can we just rename these tasks? This is so confusing. I'd recommend `unit-tests-airflow-integration` and `integration-tests-airflow-integration` or something along those lines.

Reply:
👍

Reply:
I prefixed the unit tests jobs with `unit-`. I think we can leave the integration test CI jobs as is with the format `integration-test-<integration>` as it's clear the function of the job. I feel `integration-test-<integration>-integration` becomes to wordy and there are sooo many _integration_ words in the job it loses meaning.

Reply:
I agree with the latter statement- that's why I actually suggested `integration-tests-airflow` and simply drop the last `integration` from the suffix